lib/checkout.rb

```diff
@@ -27,8 +37,19 @@ def self.marc_value(record, marc, subfield)
     nil
   end
 
+  def self.map_item_types_to_coarse_item_types(item_type)
```

Comment:
Should these nouns be plural? (shouldn't it just be `map_item_type_to_coarse_item_type`?) because it takes one and returns one.

Reply:
Fixed!